src/Pages/ResumeStorage/ResumeStorage.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import StorageButton from './Components/StorageButton';
+import FileButton from './Components/FileButton';
+import styled from 'styled-components';
+import theme from '../../Styles/theme';
+
+const ResumeStorage = () => {
+  return (
+    <StorageContainer>
```

Comment:
```suggestion
    <Container>
```
기존 방식처럼 써주셔도 괜찮은데 이름 겹칠일이 없기 때문에 간단히 써주는 것도 나쁘지않아보입니다.

Reply:
수정하였습니다!